### PR TITLE
Adding EnhancedEmergencyServiceDisclaimer

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/9273

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by dariomws — APPROVED** [review]

File: skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md [hunk 1]
```diff
@@ -175,6 +176,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnhancedEmergencyServiceDisclaimer
+Allows the tenant administrator to configure a text string, which is shown at the top of the Calls app. The user can acknowledge the string by clicking on OK. The string will be shown on client restart.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsEmergencyCallingPolicy.md:180]:
```suggestion
Allows the tenant administrator to configure a text string, which is shown at the top of the Calls app. The user can acknowledge the string by selecting OK. The string will be shown on client restart.
```

File: skype/skype-ps/skype/Set-CsTeamsEmergencyCallingPolicy.md [hunk 2]
```diff
@@ -183,6 +185,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnhancedEmergencyServiceDisclaimer
+Allows the tenant administrator to configure a text string, which is shown at the top of the Calls app. The user can acknowledge the string by clicking on OK. The string will be shown on client restart.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsEmergencyCallingPolicy.md:189]:
```suggestion
Allows the tenant administrator to configure a text string, which is shown at the top of the Calls app. The user can acknowledge the string by selecting OK. The string will be shown on client restart.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**